webapi/src/test/scala/org/knora/webapi/TestContainers.scala

```diff
@@ -64,18 +64,22 @@ object TestContainers {
   val sipiIp: String = SipiContainer.getHost
   val sipiPort: Int = SipiContainer.getFirstMappedPort
 
-  val RedisImageName: DockerImageName = DockerImageName.parse("redis:5")
-  val RedisContainer = new GenericContainer(RedisImageName)
-  RedisContainer.withExposedPorts(6379)
-  RedisContainer.start()
+
+  // The new default is the inmem cache implementation, so no need
```

Comment:
how the generation of client test data is supposed to work without Redis? 
> Client test data is generated as a side effect of running Knora's E2E tests. E2E tests use `ClientTestDataCollector` to collect test API requests and responses. The implementation of `ClientTestDataCollector` collects these in a Redis hash.

Reply:
The generation of client test data uses a different redis instance, which is started from the Makefile target.